src/components/wizard/WizardStep.tsx

```diff
@@ -118,7 +118,7 @@ export default asBaseComponent<WizardStepProps>(WizardStep);
 const styles = StyleSheet.create({
   connector: {
     borderWidth: 0.5,
-    borderColor: Colors.grey60
+    borderColor: Colors.$outlineDisabled
```

Comment:
Was it changed to grey50 so you're using the $outlineDisabled token?

Reply:
See other comments :)